docs/development/install.rst

```diff
@@ -184,3 +184,37 @@ For building this documentation,
 see :doc:`docs`.
 
 And for setting up for the front end development, see :doc:`standards`.
+
+Troubleshooting
+---------------
+
+You might face issues while installing.
```

Comment:
Can we be more explicit here and say where and when exactly this issue might occur?

Reply:
Could you read the line described as `Error` below? In a Troubleshooting section in a document, you should be less explicit in first one line. Though you should describe `error`.

Reply:
`Error: `SocialApp matching query does not exist.` this error does not occur while installing. So, I was saying It we can tell users that it may occur when they try to login through `GitHub/GitLab/BitBucket`